packages/bot-web-ui/src/stores/transactions-store.js

```diff
@@ -213,13 +213,17 @@ export default class TransactionsStore {
         const { ws, core } = this.root_store;
         const positions = core.portfolio.positions;
 
-        ws.authorized.subscribeProposalOpenContract(contract_id, response => {
-            this.is_called_proposal_open_contract = true;
-            if (!response.error) {
-                const { proposal_open_contract } = response;
-                this.updateResultsCompletedContract(proposal_open_contract);
-            }
-        });
+        // TODO: the idea is to remove the POC calls completely
+        // but adding this check to prevent making POC calls only for bot as of no
+        if (!isBot()) {
+            ws.authorized.subscribeProposalOpenContract(contract_id, response => {
+                this.is_called_proposal_open_contract = true;
+                if (!response.error) {
+                    const { proposal_open_contract } = response;
+                    this.updateResultsCompletedContract(proposal_open_contract);
+                }
+            });
+        }
```

Comment:
If the user is on bot platform the core POC should not be called, it is causing duplicate calls.